src/Avalonia.Input/KeyGesture.cs

```diff
@@ -108,19 +109,43 @@ public static KeyGesture Parse(string gesture)
 
         public override string ToString()
         {
-            var parts = new List<string>();
+            var s = new StringBuilder();
 
-            foreach (var flag in Enum.GetValues(typeof(KeyModifiers)).Cast<KeyModifiers>())
+            static void Plus(StringBuilder s)
             {
-                if (KeyModifiers.HasFlag(flag) && flag != KeyModifiers.None)
+                if (s.Length > 0)
                 {
-                    parts.Add(flag.ToString());
+                    s.Append("+");
                 }
             }
 
-            parts.Add(Key.ToString());
+            if (KeyModifiers.HasFlagCustom(KeyModifiers.Control))
+            {
+                s.Append("Ctrl");
+            }
+
+            if (KeyModifiers.HasFlagCustom(KeyModifiers.Shift))
+            {
+                Plus(s);
+                s.Append("Shift");
+            }
+            
+            if (KeyModifiers.HasFlagCustom(KeyModifiers.Alt))
+            {
+                Plus(s);
+                s.Append("Alt");
+            }
+            
+            if (KeyModifiers.HasFlagCustom(KeyModifiers.Meta))
+            {
+                Plus(s);
+                s.Append("⌘");
```

Comment:
I guess this is platform dependent, on Windows this should be windows key.

Reply:
Hmm, you're right. I don't really want to add platform-specific code to a `ToString` method, that feels like asking for trouble. Should we really be mapping the Windows key to Meta? Is there precedent for this? Feels like this should be a different key modifier to me.

Reply:
There is no "windows" key in Qt and GTK. Just Meta/Super

Reply:
Ok, so we want to follow Qt/GTK here is what you're saying (rather than make it a separate key in Avalonia)?

If so, will we need to format this text differently depending on platform, meaning simply calling `ToString()` isn't a good idea. Which means that maybe having `InputGestureText` as a string isn't a good idea.